Sources/SwiftDocC/Infrastructure/Topic Graph/AutomaticCuration.swift

```diff
@@ -230,6 +231,8 @@ extension AutomaticCuration {
     /// Add a symbol kind to `KindIdentifier.noPageKinds` if it should not generate a page in the
     /// documentation hierarchy.
     static let groupKindOrder: [SymbolGraph.Symbol.KindIdentifier] = [
+        .namespace,
```

Comment:
Why should this C++ symbol be ordered first? And does the whitespace imply you're going to add more C++ symbol kinds here in a new group?